unlock-app/src/__tests__/utils/duration.test.js

```diff
@@ -1,4 +1,4 @@
-import {durationsAsTextFromSeconds, durations, secondsAsDays} from '../../utils/durations'
+import { durationsAsTextFromSeconds, durations, secondsAsDays } from '../../utils/durations'
```

Comment:
nit! I think we use a space after brackets everywhere else? Lets maybe add a eslint rule about that.